index.html

```diff
@@ -653,59 +653,59 @@ <h2>
       </p>
 
       <p>
-Some examples contain terms (both property names and values) that are not
-defined in this specification, for illustrative purposes. These are indicated
-with a comment (<code>// external (property name|value)</code>). Such terms,
-when used in a <a>DID document</a>, are expected to be registered in the DID
-Specification Registries [[?DID-SPEC-REGISTRIES]] with links to both a formal
-definition and JSON-LD context.
+Some examples contain terms, both property names and values, that are not
+defined in this specification. These are indicated with a comment (<code>//
+external (property name|value)</code>). Such terms, when used in a <a>DID
+document</a>, are expected to be registered in the DID Specification Registries
+[[?DID-SPEC-REGISTRIES]] with links to both a formal definition and JSON-LD
+context.
       </p>
 
       <p>
-Interoperability of implementations for <a>DIDs</a> and <a>DID documents</a> will be tested by
-evaluating an implementation's ability to create and parse <a>DIDs</a> and
-<a>DID documents</a> that conform to the specification. Interoperability for
-producers and consumers of <a>DIDs</a> and <a>DID documents</a> is provided by
-ensuring the <a>DIDs</a> and <a>DID documents</a> conform. Interoperability for
-<a>DID method</a> specifications is provided by the details in each <a>DID method</a>
-specification. It is understood that, in the same way that a web browser is not
-required to implement all known <a>URI</a> schemes, conformant software that works with
-<a>DIDs</a> is not required to implement all known <a>DID methods</a> (however, all
-implementations of a given <a>DID method</a> must be interoperable for that method).
+Interoperability of implementations for <a>DIDs</a> and <a>DID documents</a> are
+tested by evaluating an implementation's ability to create and parse <a>DIDs</a>
+and <a>DID documents</a> that conform to this specification. Interoperability
+for producers and consumers of <a>DIDs</a> and <a>DID documents</a> is provided
+by ensuring the <a>DIDs</a> and <a>DID documents</a> conform. Interoperability
+for <a>DID method</a> specifications is provided by the details in each <a>DID
+method</a> specification. It is understood that, in the same way that a web
+browser is not required to implement all known <a>URI</a> schemes, conformant
+software that works with <a>DIDs</a> is not required to implement all known
+<a>DID methods</a>. However, all implementations of a given <a>DID method</a>
+are expected to be interoperable for that method.
       </p>
 
      <p>
-A <dfn>conforming DID</dfn> is any concrete expression of the rules specified
-in Section <a href="#identifier"></a> which complies with relevant normative
```

Comment:
uh oh, I'd been putting "Section" back in when I found it missing because it seemed to be present in the majority of cases. Is there editorial consensus that we're *not* using "Section" now? (I don't mind either way, just want to confirm)

Reply:
Correct, we're NOT using "Section" now. I probably had said long ago to do it because ReSpec used to not put the § marker before the number. It does this now, and I checked multiple style guides and every one I checked said that "Section §" is redundant and should be avoided.